modules/auxiliary/admin/mssql/mssql_escalate_dbowner.rb

```diff
@@ -2,10 +2,10 @@
 # This module requires Metasploit: https://metasploit.com/download
 # Current source: https://github.com/rapid7/metasploit-framework
 ##
-
+require 'metasploit/framework/mssql/client'
 
 class MetasploitModule < Msf::Auxiliary
-  include Msf::Exploit::Remote::MSSQL
+  include Metasploit::Framework::MSSQL::Client
```

Comment:
Similar to the above; We'll want to still have a MSSQL client mixin, which knows how to instantiate a Client